acceptance/catalog_services_test.go

```diff
@@ -109,4 +114,23 @@ var _ = Describe("Catalog Services", func() {
 			unbindAppService(appName, serviceName, org)
 		})
 	})
+
+	Describe("service", func() {
+		BeforeEach(func() {
+			makeCatalogService(serviceName)
+		})
+
+		It("it shows service details", func() {
+			out, err := Carrier("service "+serviceName, "")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(MatchRegexp("Service Details"))
+			Expect(out).To(MatchRegexp(`Status .*\|.* Provisioned`))
+			Expect(out).To(MatchRegexp(`Class .*\|.* mariadb`))
+			Expect(out).To(MatchRegexp(`Plan .*\|.* 10-3-22`))
```

Comment:
How stable are the plans? Can we expect the number to be there for quite some time? Maybe we should do a regex for the numbers to keep it stable?

Reply:
If the number is wrong the `create-service` will already break, because that is the plan we are asking for.
The check here simply verifies that the plan we asked for is the plan we got.